src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -500,6 +500,8 @@ uint IdealLoopTree::estimate_peeling(PhaseIdealLoop *phase) {
       if (!is_member(phase->get_loop(ctrl)) && is_loop_exit(test)) {
         return estimate;    // Found reason to peel!
       }
+    } else if (test->Opcode() == Op_ScopedValueGetResult && !is_member(phase->get_loop(phase->get_ctrl((test->as_ScopedValueGetResult())->scoped_value())))) {
```

Comment:
This line is a bit difficult to read :sweat_smile:

Reply:
Is there not any utility method for checking if a node has a ctrl that is in a loop which is inside "this" loop?